pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -287,8 +295,9 @@ func (r *InferenceServiceReconciler) updateStatus(desiredService *v1beta1api.Inf
 		// If there was a difference and there was no error.
 		isReady := inferenceServiceReadiness(desiredService.Status)
 		if wasReady && !isReady { // Moved to NotReady State
+			msg := r.GetFailConditions(desiredService)
 			r.Recorder.Eventf(desiredService, v1.EventTypeWarning, string(InferenceServiceNotReadyState),
-				fmt.Sprintf("InferenceService [%v] is no longer Ready", desiredService.GetName()))
+				fmt.Sprintf("InferenceService [%v] is no longer Ready because of: %v", desiredService.GetName(), msg))
```

Comment:
same here.

Reply:
Okay, thank you for your suggestion. We will make the adjustments.